File: rot13.js
function rot13(str) {
    let newWord = ""
     let alpha = 
     'abcdefghijklmnopqrstuvwxyzabcdefghijklmABCDEFGHIJKLMNOPQRSTUVWXYZABCDEFGHIJKLM';
     for (let i in str){
      if (str[i]){
        char = str[i]
        newChar = char.replace(/[a-z]/gi, x => alpha[alpha.indexOf(x) + 13])
        newWord += newChar
        }
     }
      return newWord
   }

console.log(rot13("I really like fast cars"))
console.log(rot13("EBG13 rknzcyr."))
console.log(rot13("Guvf vf npghnyyl gur svefg xngn V rire znqr. Gunaxf sbe svavfuvat vg! :)"))

File: ROT_13.py
def rot13(message):
  import re
  alpha = 'abcdefghijklmnopqrstuvwxyzabcdefghijklmnoABCDEFGHIJKLMNOPQRSTUVWXYZABCDEFGHIJKLMNO'
  resLst = []
  for c in message:
    if re.match(r'[a-z]|[A-Z]', c):
      newChar = alpha[alpha.index(c)+13]
      resLst.append(newChar)
    else:
      resLst.append(c)
  return ''.join(resLst)


print(rot13("I really like fast cars"))
print(rot13("EBG13 rknzcyr."))
print(rot13("Guvf vf npghnyyl gur svefg xngn V rire znqr. Gunaxf sbe svavfuvat vg! :)"))